### PR TITLE
Spec/disable eth1

### PR DESCRIPTION
**Pull request by qqqeck**
- [x] Delete Unused fields from block
  - Eth1Data
  - Deposit
  - BLSToExecution
- [x] Delete Unused fields from state
  - eth1_data
  - eth1_data_votes
  - eth1_deposit_index
  - deposit_request_start_index
- [x] Make it work
- [x] Clean up electra operation code according to spec
- [x] Delete unused prefixes
- [x] Disable POWChain
- [x] Remove unused codes related to
  - Legacy deposit
  - BLSToExecutionChange
  - Eth1Chain
- [x] (Optional) Add execution_request field for previous(<electra) blocks and add corresponding functions
- [ ] Fix tests

### CODE REVIEW

File: testing/util/altair.go [hunk 1]
```diff
@@ -80,15 +80,11 @@ func processPreGenesisDeposits(
...
-	}
-
+func buildGenesisBeaconState(genesisTime uint64, preState state.BeaconState) (state.BeaconState, error) {
 	randaoMixes := make([][]byte, params.BeaconConfig().EpochsPerHistoricalVector)
 	for i := 0; i < len(randaoMixes); i++ {
 		h := make([]byte, 32)
-		copy(h, eth1Data.BlockHash)
+		copy(h, []byte{0x01})
 		randaoMixes[i] = h
 	}
```
Comment (sb-anderson) [inline testing/util/altair.go:80]: Is it ok to use 0x01 here?

**Comment by sb-anderson** [comment]
Should remove `bls_to_execution_changes` handlings

File: beacon-chain/core/blocks/header.go [hunk 2]
```diff
@@ -119,7 +121,10 @@ func ProcessBlockHeaderNoVerify(
 	if err != nil {
 		return nil, err
 	}
-
+	log.WithFields(logrus.Fields{
+		"ProcessBlockHeaderNoVerify_parentHeader":     parentHeader,
+		"ProcessBlockHeaderNoVerify_parentHeaderRoot": hex.EncodeToString(parentHeaderRoot[:]),
+	}).Info("000000000000000000000")
```
Comment (gazzua) [inline beacon-chain/core/blocks/header.go]: This log seems unnecessary and should be removed

File: beacon-chain/core/electra/deposits.go [hunk 3]
```diff
@@ -508,8 +434,7 @@ func GetValidatorFromDeposit(pubKey []byte, withdrawalCredentials []byte, amount
 		EffectiveBalance:           0,
 		PrincipalBalance:           0,
 	}
-	maxEffectiveBalance := helpers.ValidatorMaxEffectiveBalance(validator)
-	validator.EffectiveBalance = min(amount-(amount%params.BeaconConfig().EffectiveBalanceIncrement), maxEffectiveBalance)
+	validator.EffectiveBalance = min(amount-(amount%params.BeaconConfig().EffectiveBalanceIncrement), params.BeaconConfig().MaxEffectiveBalanceAlpaca)
```
Comment (gazzua) [inline beacon-chain/core/electra/deposits.go:436]: spec in the comments might also need to be updated.
`max_effective_balance` -> `MAX_EFFECTIVE_BALANCE_ALPACA`

**Comment by qqqeck** [comment]
Closed due to change of plan.

**Pull request closed without merging**